docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -153,7 +153,8 @@ Create a container
                "CapDrop": ["MKNOD"],
                "RestartPolicy": { "Name": "", "MaximumRetryCount": 0 },
                "NetworkMode": "bridge",
-               "Devices": []
+               "Devices": [],
+               "TmpfsMounts": []
```

Comment:
Shouldn't this be in 1.17? Also description in hostconfig section is needed. Also I remember that somewhere should be "what's new" section.